pulseaudio/patches/pulseaudio-add-qubesdb-client-ifdef.patch

```diff
@@ -0,0 +1,28 @@
+--- a/src/modules/module-vchan-sink.c	2020-05-27 18:06:41.683862501 +0300
```

Comment:
Please apply this to https://github.com/QubesOS/qubes-gui-agent-linux/ directly, and add relevant `-D` flag to the makefile there. It will be much easier to maintain this way, instead of refreshing this patch each time module-vchan-sink.c changes.